clients/admin-ui/src/features/dataset/EditDatasetForm.tsx

```diff
@@ -49,18 +40,14 @@ const EditDatasetForm = ({ values, onSubmit }: Props) => {
             label="Name"
             tooltip={DATASET.name.tooltip}
             data-testid="name-input"
+            variant="block"
           />
           <CustomTextInput
             name="description"
             label="Description"
             tooltip={DATASET.description.tooltip}
             data-testid="description-input"
-          />
-          <DataCategoryInput
-            dataCategories={allDataCategories}
-            checked={checkedDataCategories}
-            onChecked={setCheckedDataCategories}
-            tooltip={DATASET.data_categories.tooltip}
+            variant="block"
```

Comment:
Talked with jack about this. We're removing the data categories selector for datasets and collections, only fields should show this.